spotbugsTestCases/src/java/ghIssues/issue574/BypassFileNotFoundExceptionExtension.java

```diff
@@ -0,0 +1,30 @@
+package ghIssues;
+
+import java.io.FileNotFoundException;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.TestExecutionExceptionHandler;
+
+
+/**
+ * Creates test extension for the @RegisterExtension test for Issue 574 on GitHub
+ * @see <a href="https://github.com/spotbugs/spotbugs/issues/574">GitHub issue</a>
+ */
+
+public class BypassFileNotFoundExceptionExtension implements TestExecutionExceptionHandler {
+
+    /**
+     * Basic extension implementation for @RegisterExtension test.
+     * @param context
+     * @param throwable
+     * @throws Throwable
+     */
+    @Override
+    public void handleTestExecutionException(ExtensionContext context, Throwable throwable) throws Throwable {
+        if (throwable instanceof FileNotFoundException) {
+            return;
+        }
+        throw throwable;
```

Comment:
Please do not use `TestExecutionExceptionHandler` for this purpose, because we cannot judge which `FileNotFoundException` is acceptable.

To keep our code readable and intuitive, use [assertThrows()](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html#assertThrows(java.lang.Class,org.junit.jupiter.api.function.Executable)) method instead.

Reply:
Or I guess that we can remove this code from this PR. Current code does not apply `Issue574RegisterExtension` during JUnit test executions.